example_unified_tiering_invocation.r

```diff
@@ -0,0 +1,11 @@
+{
+   "rule-engine-instance-name": "irods_rule_engine_plugin-unified_storage_tiering-instance",
+   "rule-engine-operation": "irods_policy_schedule_storage_tiering",
+   "delay-parameters": "<INST_NAME>irods_rule_engine_plugin-unified_storage_tiering-instance</INST_NAME><PLUSET>1s</PLUSET><EF>1h DOUBLE UNTIL SUCCESS OR 6 TIMES</EF>",
+   "storage-tier-groups": [
+       "example_group_g2",
```

Comment:
you want both groups here by default?

Reply:
yes, this is for testing.

Reply:
gotcha

Reply:
wrt apply_ vs update_, the collections themselves are not annotated with access time just the objects.  we are not updating an access time for a collection.